hazelcast/src/main/java/com/hazelcast/map/impl/operation/AbstractMultipleEntryOperation.java

```diff
@@ -67,6 +67,15 @@ protected AbstractMultipleEntryOperation(String name, EntryBackupProcessor backu
         this.backupProcessor = backupProcessor;
     }
 
+
+    @Override
+    public void beforeRun() throws Exception {
+        super.beforeRun();
+        if (mapContainer.isWanReplicationEnabled()) {
+            mapContainer.checkWanReplicationQueues();
+        }
```

Comment:
But we need to make this check only in mutating operations. Otherwise, for example a get operation  will also throw exception when WAN queue is full.